beets/autotag/match.py

```diff
@@ -241,8 +241,10 @@ def distance(items, album_info, mapping):
         dist.tracks[track] = track_distance(item, track, album_info.va)
         dist.add('tracks', dist.tracks[track].distance)
 
+    count_ign = len([x.ignorable for x in album_info.tracks if x.ignorable])
```

Comment:
OK, can you please look into it to find out why this is happening? It looks like some of the values are None, suggesting that the TrackInfo objects had `ignorable` set to `None` somewhere.